examples/moses-pln-synergy/scm/README.md

```diff
@@ -164,6 +179,10 @@ $5 = (SetLink
    )
 )
 
+### (5) Build the tautology that if X takes treatment-1, then
+    treatment-1 contains compound-A. Semi-formally
+    (\x take(x, treatment-1)) -> (\x contain(treatment-1, compound-A))
```

Comment:
well, this is still wrong; one is x, the other is y; you are using the variable x twice, which is just confusing.  These cannot be the same x, yet you use the same symbol, twice.